hello.rb

```diff
@@ -93,15 +94,17 @@ def bc_api_working?
   email = auth[:info][:email]
   store_hash = auth[:extra][:raw_info][:context].split('/')[1]
   token = auth[:credentials][:token].token
+  scope = params[:scope]
```

Comment:
Looks like this is available in the omniauth info: https://github.com/bigcommerce/omniauth-bigcommerce/blob/master/lib/omniauth/strategies/bigcommerce.rb#L39